cpp/ps0002/main (2).cpp

```diff
@@ -1,9 +1,18 @@
-short Population = '20';
-short Medicine='5';
-short Group members  ='5';
-float the price = '1.5';
-char first_name = 'A';
-int children ='50000';
+#include <iostream>
 
-float Electricity bill = '70.50';
-double Age of people ='100.123456789';
\ No newline at end of file
+int main()
+{
+    unsigned short population = 20; // 0.5/0.75
+    unsigned short medicine = 5; // 0.75/0.75
+    unsigned short group_members = 5; // 0.75/0.75
+    float pie_price = 1.5; // 0.5/0.75
+    char first_name = 'A'; // 0.75/0.75
+    unsigned int children = 50000; // 0.75/0.75
+    long double division = 0.4643765654; // 0/0.75
+    float electricity_bill = 70.50; // 0.25/0.75
+    unsigned int age_of_people = 2123456789; // 0.25/0.75
+    // total 4.5/6.75
+  
+    return 0;
+}
+// good work 👍 6.5/10
```

Comment:
- [x] program is in the correct path
- [ ] no syntax errors
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code